libraries/AP_Proximity/AP_Proximity_Backend.cpp

```diff
@@ -198,7 +198,8 @@ void AP_Proximity_Backend::database_push(float angle, float distance)
 
 // update Object Avoidance database with Earth-frame point
 // pitch can be optionally provided if needed
-void AP_Proximity_Backend::database_push(float angle, float pitch, float distance, uint32_t timestamp_ms, const Vector3f &current_pos, const Matrix3f &body_to_ned)
+// id = 65535 if the object can't be tracked
```

Comment:
maybe mentioned "UINT16_MAX" in this comment instead of "65535".  Theoretically you could default the argument as well but I don't feel strongly about this.